msal/src/main/java/com/microsoft/identity/client/TokenParameters.java

```diff
@@ -185,6 +211,18 @@ public B withScopes(List<String> scopes) {
             return self();
         }
 
+        public B withExtraOptions(List<Pair<String, String>> options) {
+            mExtraOptions = options == null ? options : new ArrayList<Pair<String, String>>(options);
+            return self();
+        }
+
+        public B withAuthorizationQueryStringParameters(
```

Comment:
Looks like this is now base class `TokenParameters` which means it will also be on the `SilentTokenParameters` - in that case this name that contains `authorization` in it would seem strange to be a parameter for a silent token request.

Reply:
I can shift it back up, honestly.  It just gets dropped off of requests if it isn't there - it was a legacy of me thinking I might use it for triggering, and then I thought better of it and wasn't sure whether we should move it back.